constants.js

```diff
@@ -5,6 +5,7 @@ const folderMapping = {
   react: 'React',
   reactnative: 'React Native',
   angular: 'Angular',
+  clientjs: 'client JS'
```

Comment:
Can we capitalize the name to `Client JS`?

Reply:
Yep